vendor.conf

```diff
@@ -4,7 +4,7 @@ github.com/boltdb/bolt e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/containerd/cgroups fe281dd265766145e943a034aa41086474ea6130
 github.com/containerd/console cb7008ab3d8359b78c5f464cb7cf160107ad5925
-github.com/containerd/containerd 1e8b09cfc6825f7e6349884b5f76e86c1f04a5d4
+github.com/containerd/containerd 5b1f69be8abdb1c44ef744f031012bfba4544536
```

Comment:
We have to keep the vendor packages in containerd/cri/vendor.conf in sync with containerd/containerd/vendor.conf. So, when updating to containerd 5b1 we also need to update the common vendor package changes that happened in containerd/containerd vendor.conf as of 5b1.

Specifically:
```
github.com/containerd/continuity d3c23511c1bf5851696cba83143d9cbcd666869b
github.com/golang/protobuf v1.1.0
google.golang.org/grpc v1.12.0
github.com/Microsoft/go-winio v0.4.7
github.com/Microsoft/hcsshim v0.6.11
```

Cheers, Mike

Reply:
Thanks, updated.

(Can we test this on CI?)

Reply:
looks like ci passed :-)

Reply:
We original enforced it, but now it is not required. We have a tool `hack/sync-vendor.sh` to manual validate.